grails-app/controllers/org/pih/warehouse/api/StockTransferApiController.groovy

```diff
@@ -187,8 +183,8 @@ class StockTransferApiController {
             }
 
             // For inbound returns
-            Date expirationDate = stockTransferItemMap.expirationDate && stockTransferItemMap.expirationDate != JSONObject.NULL ? Constants.EXPIRATION_DATE_FORMATTER.parse(stockTransferItemMap.expirationDate) : null
-            String lotNumber = stockTransferItemMap.lotNumber && stockTransferItemMap.lotNumber != JSONObject.NULL ? stockTransferItemMap.lotNumber : null
```

Comment:
the `JSONObject.NULL` appears to be not supported anymore by grails 3 so every time we encountered this code it threw an error